.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -51,4 +51,4 @@ about: Report an issue with ESLint or rules bundled with ESLint
 
 **What did you expect to happen?**
 
-**What actually happened? Please include the actual, raw output from ESLint.**
\ No newline at end of file
+**What actually happened? Please include the actual, raw output from ESLint.**
```

Comment:
Should we remove these files if we're planning to use the version with the emoji instead?

Reply:
This is my mistake -- I didn't realize the GitHub UI for this would create these extra files. I'll reset to use the original files.